File: 3566-find-the-sequence-of-strings-appeared-on-the-screen/find-the-sequence-of-strings-appeared-on-the-screen.java
class Solution {
    public List<String> stringSequence(String target) {
        List<String> ans = new ArrayList<>();
        String temp = "";
        for (int ind = 0; ind < target.length(); ++ind) {
            for (char i = 'a'; i <= target.charAt(ind); ++i) {
                temp += i;
                ans.add(temp);
                temp = temp.substring(0, temp.length() - 1);
            }
            temp += target.charAt(ind);
        }
        return ans;
    }
}

File: 3566-find-the-sequence-of-strings-appeared-on-the-screen/find-the-sequence-of-strings-appeared-on-the-screen.py
class Solution:
    def stringSequence(self, target: str) -> List[str]:
        ans=[]
        temp=""
        for ind in range(len(target)):
            for i in range(ord('a'),ord(target[ind])+1):
                temp+=chr(i)
                ans.append(temp)
                temp=temp[:-1]
            temp+=target[ind]
        return ans